src/utility.js

```diff
@@ -53,14 +53,15 @@ function parseUncssrc(filename) {
 function parsePaths(source, stylesheets, options) {
     return stylesheets.map(function (sheet) {
         var sourceProtocol;
+        var isLocalFile = sheet.substr(0, 5) === 'file:';
 
         if (sheet.substr(0, 4) === 'http') {
             /* No need to parse, it's already a valid path */
             return sheet;
         }
 
         /* Check if we are fetching over http(s) */
-        if (isURL(source)) {
+        if (isURL(source) && !isLocalFile) {
             sourceProtocol = url.parse(source).protocol;
```

Comment:
We have a line here that computes the url's protocol. Can this be used further up above to check for both 'file' and 'http' and simplify things?